invenio_app/config.py

```diff
@@ -75,11 +81,18 @@
     'session_cookie_secure': True,
     'session_cookie_http_only': True
 }
-"""Default Secure Headers.
+"""Talisman default Secure Headers configuration.
+
+As default, invenio assumes that HTTPS is enabled.
+If you are not using SSL, then it's required to manually disable them
+overwriting the configurations related to it.
```

Comment:
This sentence is a bit confusing (I'm not sure if you should disable the Invenio settings that are overwriting Talisman configuration, or if we should disable the Talisman configuration with the Invenio config).
This can be simplified to (if I understood the sentence correctly :P) something like that:
`If you are not using SSL, then remember to disable the configuration options related to HTTPS`
BTW, should be explicitly say which configuration options from this list should be changed (all of them or only some)?